ehrql/tables/raw/tpp.py

```diff
@@ -509,3 +510,234 @@ class apcs_cost_historical(EventFrame):
     discharge_date = Series(
         datetime.date,
     )
+
+
+@table
+class covid_therapeutics_raw(EventFrame):
+    """
+    The COVID Therapeutics dataset contains information on COVID treatments used in inpatient
+    and outpatient settings.
+
+    **Metadata**
+
+    * **Data provider** NHS England
+    * **Participation / Coverage** Inpatients and outpatients treated with antivirals/nMABs for COVID-19 in England
+    * **Provenance** Data sourced largely from BlueTeq system (forms completed by clinicians)
+    * **Update frequency in OpenSAFELY** Approximately weekly
+    * **Delay between event occurring and event appearing in OpenSAFELY** Approximately 2-9 days
+    * **Collected information** Treatment start date; therapeutic intervention; COVID indication, current status, risk group, region
+
+
+    **Overview**
+
+    Antivirals and neutralising monoclonal antibodies (nMABs) for COVID-19 can be
+    administered in inpatient setting or, for outpatients, in COVID Medicine Delivery
+    Units (CMDUs) specifically set up for this purpose. For patients considered for
+    these treatments, clinicians submit completed forms to NHS England. Each row
+    represents one completed form for one course of treatment. Data received by
```

Comment:
It occurs to me that this isn't consistent with the description of the `count` field below being "number of forms". What would a count of 2 mean, if a row represents one completed form?

Reply:
That's a perfectly reasonable question! I don't know who we'd chase that up with though.